src/animator.js

```diff
@@ -239,8 +242,13 @@ export class VelocityAnimator {
    * @returns Resolved when the animation is done
    */
   removeClass(element: HTMLElement, className: string): Promise<boolean> {
```

Comment:
I'm no TypeScript expert. However, `_runElementAnimation()` returns `Promise<any>`, so then it could just as well be `Promise<any>` for `removeClass()`.  But wouldn't that violate the Animator interface definition, which has `Promise<bool>`? Should I perhaps add a `.then()` that returns a boolean on success?